test/actor_test.py

```diff
@@ -2233,17 +2234,19 @@ def kill_actor(actor):
         """Kill actor process."""
         pid = ray.get(actor.get_pid.remote())
         os.kill(pid, signal.SIGKILL)
-        time.sleep(1)
 
     actor = ReconstructableActor.remote()
     # Call increase 3 times
     for _ in range(3):
-        ray.get(actor.increase.remote())
-    # kill actor process
+        actor.increase.remote()
+    # Call increase again with some delay. So we can kill the actor while
+    # the actor is still executing this task.
+    actor.increase.remote(delay=2.0)
+    # Kill actor process
```

Comment:
This doesn't actually kill the actor while it is executing the task, because `kill_actor` calls `ray.get(actor.get_pid.remote())`, which will wait until `actor.increase` has finished running.